shell/client/admin/user-details-client.js

```diff
@@ -231,17 +256,22 @@ Template.newAdminUserDetails.events({
     });
   },
 
-  "click .delete-account"(evt) {
-    const instance = Template.instance();
-    const confirm = window.confirm("Really delete this account forever? This operation is permament and irreversible.");
-    if (confirm) {
-      Meteor.call("deleteAccount", instance.userId, function (err) {
-        if (err) {
-          window.alert(err);
-        } else {
-          Router.go("newAdminUsers");
-        }
-      });
-    }
+  "click [name=\"delete-account\"]"(evt, instance) {
+    instance.showDeletePopup.set(true);
```

Comment:
Also clear the delete error here, lest you see the one from the dialog you previously dismissed?